src/components/DocumentationTopic.vue

```diff
@@ -737,6 +737,16 @@ export default {
       border-radius: var(--code-border-radius);
     }
 
+    .source:not(.has-multiple-lines) > code {
+      @include inTargetIde() {
+        white-space: pre-wrap;
+
+        .token-identifier {
```

Comment:
Why are we only breaking the `token-identifier`?

Reply:
The thought is that we don't want to break up `keywords`, or break at any character for `attribute` or `typeIdentifier` tokens when they already have word breaks in them.